gslib/wildcard_iterator.py

```diff
@@ -41,16 +46,20 @@
 from gslib.utils.text_util import FixWindowsEncodingIfNeeded
 from gslib.utils.text_util import PrintableStr
 
+if six.PY3:
+  # StandardError was removed, so use the base exception type instead
+  StandardError = Exception
+
 
 FLAT_LIST_REGEX = re.compile(r'(?P<before>.*?)\*\*(?P<after>.*)')
 
 _UNICODE_EXCEPTION_TEXT = (
-    'Invalid Unicode path encountered (%s). gsutil cannot proceed '
-    'with such files present. Please remove or rename this file and '
-    'try again. NOTE: the path printed above replaces the '
-    'problematic characters with a hex-encoded printable '
-    'representation. For more details (including how to convert to a '
-    'gsutil-compatible encoding) see `gsutil help encoding`.')
+  'Invalid Unicode path encountered (%s). gsutil cannot proceed '
```

Comment:
Please fix the spacing back to what it was before (4 spaces instead of 2)

Reply:
Fixed indentation spacing.